case.jsonld

```diff
@@ -1094,6 +1094,7 @@
                     "@id": "kb:056c4664-bcf0-54e9-a24d-0cc97e7fd429",
                     "@type": "uco-observable:FileFacet",
                     "uco-observable:fileName": "IMG_0123.jpg",
+                    "uco-observable:filePath": "file/Image/IMG_0123.jpg",
```

Comment:
This value of `filePath` is inconsistent with `files/Image/ImG_0123.jpg` later in the `PathRelationsFacet`.

Reply:
And, since this is the file _post_-extraction, `filePath` should be giving the absolute path within the file system - the examiner's workspace file system.  There might also be committee debate about what the interpretation is on whether this should pertain to the active file system hierarchy.

I suggest `filePath` be cut from this node.